src/Javinizer/Public/Get-JVData.ps1

```diff
@@ -58,6 +58,14 @@ function Get-JVData {
         [Alias('scraper.movie.dmm.scrapeactress')]
         [Boolean]$DmmScrapeActress,

```

Comment:
The parameter aliases seem to be incorrect here.

`scraper.movie.dmm.scrapeactress` => `scraper.option.dmm.scrapeactress` (This one is my mistake)
`scraper.movie.dmm.idpreference` => `scraper.option.dmm.idpreference`
`scraper.movie.dmm.r18prfeference` => `scraper.option.r18.idpreference`

Reply:
I have consolidated all of them in `scraper.option.idpreference`.